apps/backend/Dockerfile

```diff
@@ -0,0 +1,61 @@
+
+FROM node:16-alpine3.16 AS development
+
+WORKDIR /app
+
+COPY --chown=node:node package.json ./
+
+COPY --chown=node:node yarn.lock ./
+
+RUN corepack enable
+
+RUN corepack prepare yarn@3.2.3 --activate
+
+RUN yarn set version 3.2.3
+
+RUN yarn install
+
+COPY --chown=node:node . .
+
+USER node
+
+ENV PORT=3000
+
+EXPOSE 3000
+
+CMD ["yarn", "run", "start:dev"]
+
+###################
+# BUILD FOR PRODUCTION
+###################
+
+FROM node:16-alpine3.16 AS build
```

Comment:
We will run the apps as lambdas, 

but curious why the production build is in the same file as local build

Reply:
(I am not sure if I have done the incremental build properly though :)